content/reference/current/pools/kubernetes/index.md

```diff
@@ -39,20 +54,35 @@ To configure the type of instances you want to spawn, you need to fill the form
 - **Memory limit**: The maximum memory that you need for each injector
 - **Custom labels**: Optional labels to add to all injector pods, services, ingresses and routes
 - **Node selector**: An optional [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) to add to each injector pod (they will only run on nodes with matching labels) 
+- **Environment variables**: The environment variables configured for each injector
 
 Limits and requests for memory are measured in bytes. You can express memory as a plain integer or as a fixed-point integer using one of these suffixes: E, P, T, G, M, K. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki.
 
-NOTE: If your Gatling Enterprise instance belongs to a Kubernetes cluster, you don't have to provide a **Kubernetes Url** and a **Service Account Token**.
-You can still choose to configure it, for example to create a pool in another cluster, by unticking `Use local cluster`.
-Also, you can specify preferring **Internal IP** over the **External IP** for connecting to Kubernetes nodes, by ticking `Prefer internal IP`.
+## Tolerations settings
 
-{{< img src="kubernetes-local.png" alt="Kubernetes local" >}}
+To configure [torelations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/), you may fill:
+
+{{< img src="tolerations-settings.png" alt="Kubernetes credentials" >}}
+
+- **Key**: toleration key
+- **Operator**: Either exists or equal
+  - **Exists**: Any node matching the key will have the effect applied, no need to specify the value
+  - **Equal**: Any node matching the key with the given value will have the effect applied
```

Comment:
Are we sure those are the only possible values? Where does this come from?

Reply:
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/

> the operator is Exists (in which case no value should be specified), or
> the operator is Equal and the values are equal.

Reply:
It's the same for other fields (e.g: effect which is specified aswell), and the link between the fields (we should have an ADT, where value is required only when Equal is selected as an operator).

Reply:
Do we want to change this right now? Or modify later?

Reply:
Modify later 👍